docs/docs_settings.py

```diff
@@ -28,18 +28,71 @@
         FEATURES[key] = True
 
 # Settings that will fail if we enable them, and we don't need them for docs anyway.
-FEATURES['RUN_AS_ANALYTICS_SERVER_ENABLED'] = False
-FEATURES['ENABLE_SOFTWARE_SECURE_FAKE'] = False
-FEATURES['ENABLE_MKTG_SITE'] = False
+FEATURES["RUN_AS_ANALYTICS_SERVER_ENABLED"] = False
+FEATURES["ENABLE_SOFTWARE_SECURE_FAKE"] = False
+FEATURES["ENABLE_MKTG_SITE"] = False
+
+INSTALLED_APPS.extend(
+    [
+        "cms.djangoapps.contentstore.apps.ContentstoreConfig",
+        "cms.djangoapps.course_creators",
+        "cms.djangoapps.xblock_config.apps.XBlockConfig",
+        "lms.djangoapps.lti_provider",
+    ]
+)
+
+# Swagger generation details
+openapi_security_info_basic = (
+    "Obtain with a `POST` request to `/user/v1/account/login_session/`.  "
+    "If needed, copy the cookies from the response to your new call."
+)
+openapi_security_info_jwt = dedent(
+    """
+    Obtain by making a `POST` request to `/oauth2/v1/access_token`.
+
+    You will need to be logged in and have a client ID and secret already created.
 
-INSTALLED_APPS.extend([
-    'cms.djangoapps.contentstore.apps.ContentstoreConfig',
-    'cms.djangoapps.course_creators',
-    'cms.djangoapps.xblock_config.apps.XBlockConfig',
-    'lms.djangoapps.lti_provider',
-])
+    Your request should have the headers
+
+    ```
+    'Content-Type': 'application/x-www-form-urlencoded'
+    ```
+
+    Your request should have the data payload
+
+    ```
+    'grant_type': 'client_credentials'
+    'client_id': [your client ID]
+    'client_secret':  [your client secret]
+    'token_type': 'jwt'
+    ```
+
+    Your JWT will be returned in the response as `access_token`. Prefix with `JWT ` in your header.
```

Comment:
is the space in `JWT ` intentional? My assumption by reading this would be to add this to my header: 

`JWT {access token string}`